superset/views/base.py

```diff
@@ -327,6 +329,17 @@ def common_bootstrap_payload() -> Dict[str, Any]:
     }
 
 
+def get_moment_locale(locale):
```

Comment:
Feel free to create a `superset/utils/i18n.py` file and move this function over.